app/src/main/java/woowacourse/movie/domain/movieTimePolicy/WeekdayMovieTime.kt

```diff
@@ -0,0 +1,17 @@
+package woowacourse.movie.domain.movieTimePolicy
+
+import java.time.LocalDate
+import java.time.LocalTime
+
+object WeekdayMovieTime : MovieTimePolicy {
+    private val WEEKDAY = 1..5
+    private val SCREEN_TIME = 10 until 24
+    private const val SCREEN_TIME_INTERVAL = 2
+
+    override fun generateTime(date: LocalDate): List<LocalTime> {
+        if (date.dayOfWeek.value in WEEKDAY) {
+            return SCREEN_TIME.step(SCREEN_TIME_INTERVAL).map { LocalTime.of(it, 0) }
+        }
+        return listOf()
```

Comment:
```suggestion
        return emptyList()
```
빈 리스트인 경우에는 emptyList를 활용하면 좀 더 가독성이 좋아집니다. (이하 관련 내용 동일)